spec/ParseLiveQuery.spec.js

```
@@ -1308,4 +1308,59 @@
     await new Promise(resolve => setTimeout(resolve, 100));
     expect(createSpy).toHaveBeenCalledTimes(1);
   });
+
+  fit('Live query should work if needle is ParsePointer and haystack is any[], checking QueryTools.js>contains', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)